### DISCUSSION TITLE
Is it possible to add displayed information with Overclocker upgrade?

### DISCUSSION THREAD

**Discussion by Pyromanc3r** [VintageModsReforged/BlockHelperAddons#4] [Ideas]
For example, when Overclocker upgrade is installed the tooltip will say that it is already MV tier and 128 EU/t input
![2024-11-25_17 02 33](https://github.com/user-attachments/assets/0ac8217f-a94d-4a66-ace5-d9393790a17c)

**Comment by CrossVas** [comment]
> For example, when Overclocker upgrade is installed the tooltip will say that it is already MV tier and 128 EU/t input
> ![2024-11-25_17 02 33](https://github.com/user-attachments/assets/0ac8217f-a94d-4a66-ace5-d9393790a17c)
> 

Em... Try Transformer Upgrade

**Comment by Pyromanc3r** [comment]
Even if all 4 upgrades are installed, the information will not change.
E-Furnace is now taking 8192 EU/t, but information shows it is still LV Tier with 32 EU/t. 
If the intention is to focus only on the number of upgrades installed, then I understand the decision.
![2024-11-26_19 31 16](https://github.com/user-attachments/assets/457bf6c8-a396-4156-a81c-efbf2ff5e956)

**Comment by CrossVas** [comment]
Moved to #5, closing this
